File: src/ExamRegister.WebApi.Abstractions/DTO/MedicoClinicaDTO.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ExamRegister.WebApi.Abstractions.DTO
{
    public partial class MedicoClinicaDTO
    {
        public Guid idmedcli { get; set; }
        public Guid idmedico { get; set; }
        public Guid idclinica { get; set; }

        public ClinicaDTO clinica { get; set; }
    }
}

File: src/ExamRegister.UI/src/app/shared/DTOs/medicoclinica-dto.ts
import { ClinicaDTO } from "./clinica-dto";
import { MedicoDTO } from "./medico-dto";

export interface MedicoclinicaDTO {
    idmedcli: string | null;
    idmedico: string | null;
    idclinica: string;

    clinica: ClinicaDTO;
    medico?: MedicoDTO
}
